account_invoice_blocking/models/account_move.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
```

Comment:
Perhaps a more suitable name would be 
```suggestion
class AccountMove(models.Model):
```